tools/c7n_gcp/c7n_gcp/actions/labels.py

```diff
@@ -0,0 +1,138 @@
+# Copyright 2019 Microsoft Corporation
```

Comment:
guessing your not an ms employee, if your doing work on behalf of a corp, the corp name should go here, else use your individual name.

Reply:
You dont have static header for all your files?
I just copy/pasted it..

I will remove it then?

Reply:
the license header should be the same across files, but the copyright assignment should be per the person/company authoring the file.